cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala

```diff
@@ -80,9 +80,9 @@ class CallTest extends DocumentingTest {
 
     section("View the signature for a procedure", "call-view-the-signature-for-a-procedure") {
       p("""To `CALL` a procedure, its input parameters need to be known, and to use `YIELD`, its output parameters need to be known.
-          #The built-in procedure `dbms.procedures` returns the name, signature and description for all procedures.
+          #Use SHOW PROCEDURES which returns the name, signature and description for all procedures.
```

Comment:
```suggestion
          #Use `SHOW PROCEDURES` which returns the name, signature and description for all procedures.
```
maybe even rewrite as
```suggestion
          #The `SHOW PROCEDURES` command returns the name, signature and description for all procedures.
```
to be closer to the original sentence?